usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'verify', 'pack', 'repo', 'env', 'server', 'token')
usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'verify', 'env', 'server', 'repo', 'pack', 'token')
no change     /curc/sw/anaconda3/2023.09/condabin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda-env
no change     /curc/sw/anaconda3/2023.09/bin/activate
no change     /curc/sw/anaconda3/2023.09/bin/deactivate
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.sh
no change     /curc/sw/anaconda3/2023.09/etc/fish/conf.d/conda.fish
no change     /curc/sw/anaconda3/2023.09/shell/condabin/Conda.psm1
no change     /curc/sw/anaconda3/2023.09/shell/condabin/conda-hook.ps1
no change     /curc/sw/anaconda3/2023.09/lib/python3.11/site-packages/xontrib/conda.xsh
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.csh
no change     /home/mlippincott@xsede.org/.bashrc
No action taken.
Processing well_fov D11-2 for patient NF0018
Processing directory  of Segmenting Nuclei
  0%|          | 0/20 [00:00<?, ?it/s]  5%|▌         | 1/20 [00:02<00:44,  2.35s/it] 10%|█         | 2/20 [00:02<00:22,  1.27s/it] 15%|█▌        | 3/20 [00:03<00:15,  1.08it/s] 20%|██        | 4/20 [00:03<00:12,  1.29it/s] 25%|██▌       | 5/20 [00:04<00:10,  1.47it/s] 30%|███       | 6/20 [00:04<00:08,  1.59it/s] 35%|███▌      | 7/20 [00:05<00:07,  1.66it/s] 40%|████      | 8/20 [00:06<00:07,  1.71it/s] 45%|████▌     | 9/20 [00:06<00:06,  1.74it/s] 50%|█████     | 10/20 [00:07<00:05,  1.77it/s] 55%|█████▌    | 11/20 [00:07<00:05,  1.79it/s] 60%|██████    | 12/20 [00:08<00:04,  1.79it/s] 65%|██████▌   | 13/20 [00:08<00:03,  1.81it/s] 70%|███████   | 14/20 [00:09<00:03,  1.82it/s] 75%|███████▌  | 15/20 [00:09<00:02,  1.84it/s] 80%|████████  | 16/20 [00:10<00:02,  1.86it/s] 85%|████████▌ | 17/20 [00:10<00:01,  1.87it/s] 90%|█████████ | 18/20 [00:11<00:01,  1.88it/s] 95%|█████████▌| 19/20 [00:11<00:00,  1.87it/s]100%|██████████| 20/20 [00:12<00:00,  1.90it/s]100%|██████████| 20/20 [00:12<00:00,  1.60it/s]
Completed Nuclei Segmentation
Segmenting Cells
Completed Cell Segmentation
Segmenting Organoids
  0%|          | 0/25 [00:00<?, ?it/s]  8%|▊         | 2/25 [00:00<00:01, 11.73it/s] 16%|█▌        | 4/25 [00:00<00:01, 10.53it/s] 24%|██▍       | 6/25 [00:00<00:01, 11.22it/s] 32%|███▏      | 8/25 [00:00<00:01, 10.68it/s] 40%|████      | 10/25 [00:00<00:01, 10.37it/s] 48%|████▊     | 12/25 [00:01<00:01, 10.91it/s] 56%|█████▌    | 14/25 [00:01<00:01, 10.56it/s] 64%|██████▍   | 16/25 [00:01<00:00, 10.96it/s] 72%|███████▏  | 18/25 [00:01<00:00, 10.56it/s] 80%|████████  | 20/25 [00:01<00:00, 10.30it/s] 88%|████████▊ | 22/25 [00:02<00:00, 10.71it/s] 96%|█████████▌| 24/25 [00:02<00:00, 10.39it/s]100%|██████████| 25/25 [00:02<00:00, 10.57it/s]
  0%|          | 0/18 [00:00<?, ?it/s]  6%|▌         | 1/18 [00:01<00:32,  1.93s/it] 11%|█         | 2/18 [00:03<00:23,  1.46s/it] 17%|█▋        | 3/18 [00:04<00:19,  1.31s/it] 22%|██▏       | 4/18 [00:05<00:17,  1.22s/it] 28%|██▊       | 5/18 [00:07<00:20,  1.55s/it] 33%|███▎      | 6/18 [00:08<00:16,  1.39s/it] 39%|███▉      | 7/18 [00:09<00:14,  1.29s/it] 44%|████▍     | 8/18 [00:10<00:12,  1.20s/it] 50%|█████     | 9/18 [00:11<00:10,  1.15s/it] 56%|█████▌    | 10/18 [00:12<00:08,  1.11s/it] 61%|██████    | 11/18 [00:13<00:07,  1.08s/it] 67%|██████▋   | 12/18 [00:14<00:06,  1.06s/it] 72%|███████▏  | 13/18 [00:15<00:05,  1.07s/it] 78%|███████▊  | 14/18 [00:16<00:04,  1.07s/it] 83%|████████▎ | 15/18 [00:17<00:03,  1.07s/it] 89%|████████▉ | 16/18 [00:18<00:02,  1.06s/it] 94%|█████████▍| 17/18 [00:19<00:01,  1.04s/it]100%|██████████| 18/18 [00:20<00:00,  1.02s/it]100%|██████████| 18/18 [00:20<00:00,  1.16s/it]
Completed Organoid Segmentation
Decoupling nuclei
0it [00:00, ?it/s]2it [00:00, 19.96it/s]4it [00:00, 16.30it/s]6it [00:00, 15.30it/s]8it [00:00, 10.72it/s]10it [00:00, 11.23it/s]12it [00:00, 12.10it/s]14it [00:01, 12.76it/s]16it [00:01, 12.58it/s]18it [00:01, 12.46it/s]20it [00:01, 13.00it/s]21it [00:01, 13.17it/s]
Traceback (most recent call last):
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3805, in get_loc
    return self._engine.get_loc(casted_key)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "index.pyx", line 167, in pandas._libs.index.IndexEngine.get_loc
  File "index.pyx", line 196, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7081, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7089, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'label'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/4.reconstruct_3D_masks.py", line 372, in <module>
    mask[mask == tmp_df.iloc[i]["original_label"]] = tmp_df.iloc[i]["label"]
                                                     ~~~~~~~~~~~~~~^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/series.py", line 1121, in __getitem__
    return self._get_value(key)
           ^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/series.py", line 1237, in _get_value
    loc = self.index.get_loc(label)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3812, in get_loc
    raise KeyError(key) from err
KeyError: 'label'
Decoupling cell
0it [00:00, ?it/s]1it [00:00,  8.76it/s]2it [00:00,  7.56it/s]3it [00:00,  5.79it/s]4it [00:00,  5.29it/s]5it [00:00,  5.86it/s]6it [00:00,  6.18it/s]7it [00:01,  5.92it/s]8it [00:01,  5.77it/s]9it [00:01,  5.65it/s]10it [00:01,  5.53it/s]11it [00:01,  5.49it/s]12it [00:02,  5.12it/s]13it [00:02,  4.89it/s]14it [00:02,  4.73it/s]15it [00:02,  4.93it/s]16it [00:02,  5.40it/s]17it [00:03,  5.77it/s]18it [00:03,  6.08it/s]19it [00:03,  6.31it/s]21it [00:03,  8.47it/s]21it [00:03,  6.03it/s]
Decoupling organoid
0it [00:00, ?it/s]3it [00:00, 14.88it/s]5it [00:00,  9.52it/s]7it [00:00,  8.27it/s]8it [00:00,  7.88it/s]9it [00:01,  7.62it/s]10it [00:01,  7.43it/s]11it [00:01,  7.30it/s]12it [00:01,  7.19it/s]13it [00:01,  7.09it/s]14it [00:01,  7.04it/s]15it [00:01,  7.01it/s]16it [00:02,  7.00it/s]17it [00:02,  7.00it/s]18it [00:02,  6.99it/s]19it [00:02,  7.61it/s]21it [00:02,  8.17it/s]
Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/4.reconstruct_3D_masks.py", line 275, in <module>
    df["indexes"] = df["index1"].astype(str) + "-" + df["index2"].astype(str)
                    ~~^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/range.py", line 417, in get_loc
    raise KeyError(key)
KeyError: 'index1'
Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/5.create_cytoplasm_masks.py", line 74, in <module>
    ).resolve(strict=True)
      ^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/pathlib.py", line 993, in resolve
    s = os.path.realpath(self, strict=strict)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 416, in realpath
  File "<frozen posixpath>", line 451, in _joinrealpath
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/data/NF0018/processed_data/D11-2/nuclei_masks_reconstructed.tiff'
Segmentation complete
